bin/inx-chronicle/src/api/mod.rs

```diff
@@ -68,6 +68,7 @@ impl Actor for ApiWorker {
         let api_handle = cx.handle().clone();
         let port = self.config.port;
         let routes = routes()
+            .layer(Extension((&***cx).clone()))
```

Comment:
A comment would be nice with what type we end up with via this deref chain (ScopeView?)